packages/server/src/api/controllers/migrations.ts

```diff
@@ -1,14 +1,28 @@
-import { migrate as migrationImpl, MIGRATIONS } from "../../migrations"
-import { BBContext } from "@budibase/types"
+import { context } from "@budibase/backend-core"
```

Comment:
We might want to create a separate app migration API - reason being the migration system isn't totally deprecated yet - we'll probably keep it around for a little while as tenant level migrations still have to be ran this way. A separate app migration API would mean we can keep the two totally separate until we handle tenant level migrations as well.

Reply:
Yup, this is just for the PoC and to get the idea. Then things will need to be moved to their proper place